test/test_torch.py

```diff
@@ -5940,6 +5940,33 @@ def test_isinf_type(self, device):
         with self.assertRaises(TypeError):
             torch.isinf(1)  # Parameter must be a tensor
 
+    @dtypes(*(torch.testing.get_all_int_dtypes() + torch.testing.get_all_fp_dtypes(include_bfloat16=False)))
```

Comment:
This PR will need to rebased because https://github.com/pytorch/pytorch/pull/47356 moved the tests around. `msort` tests can be put into `test_sort_and_select` since it sorts its input.

Reply:
Addressed, Thanks for reminding that!